draft-ietf-httpbis-digest-headers.md

```diff
@@ -718,6 +720,9 @@ The response includes a `Content-Encoding: br` that indicates
 the selected representation is brotli encoded.
 The `Digest` field-value is therefore different compared to the request.
 
+The response body is displayed as a base64-encoded string
```

Comment:
I prefer to separate the sentence because of the `therefore` in line 721. @LPardue

Reply:
I can live with the sentence order but there is no need to split the paragraph IMO.

This is an example where I think the general document readability can be improved with another editorial pass over all examples. However, we can tackle that in a separate PR once design issues are closed out.

Reply:
Sure! We can schedule it one month before Vancouver: wdyt?

Reply:
SGTM